features/display_dishes.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Display dishes on the menu page
+    As a User,
+    In order to decide what to eat,
+    I would like to see dishes on the menu page
+
+    Background: Dish
+        Given the following dishes exists
+        | name  | price | description   |
+        | Pasta | 100   | Very nice     |   
+        When I visit the menu page
+
+    Scenario: Display dishes on the menu page
+        Then I should see 'Pasta' in the dish table
```

Comment:
Go over the scenario and background and feature and make sure they follow the right structure, `given when then`